DockerLaunchCommands.MD

```diff
@@ -0,0 +1,14 @@
+# Docker launch commands for coder
+
+## default coder uid:guid
+docker run -it -p 127.0.0.1:8443:8443 \
+ -v "${PWD}/code-server:/home/coder/project" \
+ satlus-code-server:latest --allow-http --no-auth
```

Comment:
The image name here is of yours and not the official one.

Reply:
Great catch - I'm going to remove this file, and just add a section to the readme for non-root usage. It will avoid confusion